src/types/commands.ts

```diff
@@ -168,6 +168,9 @@ export const readonlyAllowedCommands = new Set<CommandTypes>([
   "START",
   "ACTIVATE_SHEET",
 
+  // allow to check checkboxes, select dropdowns, use filters
+  "UPDATE_CELL",
```

Comment:
It seems weird - we'd rely on the fact that this will not be propagated on the network?

Reply:
yes... if you have better/more robust idea...